card/mod/05_standard/set/all/list_changes.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
+
+event :trunk_cardtype_of_a_list_relation_changed,
+      changed: :type, after: :store, on: :update,
+      when: proc { |c| Codename[:list] } do
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.